europe/README.EUROPE

```diff
@@ -294,3 +294,145 @@ Residuals at 130 further test points average 0.06m, maximum 0.16m.
+
+Deformation model used in transformation between ISN93 reference frame and ISN2016 reference frame
+Horizontal transformation is designed so that the ISN2016 points should fit within 1-2 cm accuracy, but in the area around the earthquake area in the South and North of Vatnajökull the accuracy is worse.
+Vertical transformation 
+Recommended to transform from ISN93 3D geodetic (EPSG:4945) to ISN2016 3D geodetic (EPSG:8085)
+
+See the `ISL` init-file for examples of use.
+
+* ISN93_ISN2016.gsb
+* ISN93_ISN2016.gtx
+
+
+### Iceland: Deformation model: ISN2004 -> ISN2016 
+
+*Source*: [National Land Survey of Iceland](https://atlas.lmi.is/LmiData/index.php?id=626468364600)  
+*Format*: NTv2,GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)  
+*Credit*: (c)2017-2019 Grid created by Guðmundur Valsson.
+
+Deformation model used in transformation between ISN2004 reference frame and ISN2016 reference frame
+Horizontal transformation is designed so that the ISN2016 points should fit within 1-2 cm accuracy, but in the area around the earthquake area in the South and North of Vatnajökull the accuracy is worse.
+Vertical transformation 
+Recommended to transform from ISN2004 3D geodetic (EPSG:5323) to ISN2016 3D geodetic (EPSG:8085)
+
+See the `ISL` init-file for examples of use.
+
+* ISN2004_ISN2016.gsb
+* ISN2004_ISN2016.gtx
+
+
+### Iceland: ISN2004 ellipsoidal height  -> ISH2004 vertical datum
+
+*Source*: [National Land Survey of Iceland](https://atlas.lmi.is/LmiData/index.php?id=626468364600)  
+*Format*: GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)  
+*Credit*: (c)2017-2019 Grid created by Guðmundur Valsson.
+
+Geoid created in cooperation between the National Land Survey of Iceland and DTU Space in  December 2010. 
+A gravimetric geoid was fitted to National Height System ISH2004 (EPSG:8089) with over 300 GNSS levelling points
+with ellipsoidal height in ISN2004 (EPSG:5323). The std. of the fit was 1.7 cm.
+
+Grid to convert from ISN2004 ellipsoidal height (EPSG:5323) to national vertical datum ISH2004 (EPSG:8089)
+The horizontal grid coordinates are referenced to ISN2004
+
+See the `ISL` init-file for examples of use.
+
+* Icegeoid_ISN2004.gtx 
+
+### Iceland: ISN93 ellipsoidal height  -> ISH2004 vertical datum
+
+*Source*: [National Land Survey of Iceland](https://atlas.lmi.is/LmiData/index.php?id=626468364600)  
+*Format*: GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)  
+*Credit*: (c)2017-2019 Grid created by Guðmundur Valsson.
+
+Icegeoid_ISN2004.gtx plus corrections for vertical difference between ISN93(EPSG:4945) and ISN2004 (EPSG:5323).
+
+Grid to convert from ISN93 ellipsoidal height (EPSG:4945) to national vertical datum ISH2004 (EPSG:8089)
+The horizontal grid coordinates are referenced to ISN93
+
+See the `ISL` init-file for examples of use.
+
+* Icegeoid_ISN93.gtx
```

Comment:
As far as I can see this file is not added to the repository. The only file with the "icegeoid" name is `icegeoid_2011.gtx`

Reply:
fixed in 6c8195d2843860a63b615f8c4398d7dae0fc0a59